crates/edr_evm/src/block/remote.rs

```diff
@@ -79,7 +79,7 @@ impl RemoteBlock {
             gas_used: block.gas_used,
             timestamp: block.timestamp,
             extra_data: block.extra_data,
-            mix_hash: block.mix_hash.ok_or(CreationError::MissingMixHash)?,
+            mix_hash: block.mix_hash.unwrap_or_default(),
```

Comment:
I think we should add a GitHub issue and `TODO` here to indicate that this should be reverted once we have multi-chain support.

Then we (or someone else) can add an implementation for this particular chai.

Reply:
Done, thanks.

Can you re-review this? I had to solve a merge conflict and it was pretty straightforward, but it wouldn't hurt to double-check.